Library/Homebrew/dev-cmd/generate-formula-api.rb

```diff
@@ -69,11 +77,15 @@ def generate_formula_api
           File.write("api/formula/#{name}.json", FORMULA_JSON_TEMPLATE)
           File.write("formula/#{name}.html", html_template_name)
         end
+
+        homebrew_core_tap_hash["formulae"][formula.name] = formula.to_api_hash.except("name")
       rescue
         onoe "Error while generating data for formula '#{name}'."
         raise
       end
 
+      homebrew_core_tap_json = JSON.generate(homebrew_core_tap_hash)
+      File.write("api/homebrew-core.json", homebrew_core_tap_json) unless args.dry_run?
```

Comment:
```suggestion
      File.write("api/homebrew-core-internal.json", homebrew_core_tap_json) unless args.dry_run?
```

or something, unless I've missed something here.

Reply:
It was an idea from @Bo98 that I liked. See this thread starting here https://github.com/Homebrew/brew/pull/16460#discussion_r1460722950

Reply:
Might make sense to lean on path components so it's `/api/internal/v3/homebrew-core.json`. The version being a "directory" like that is very common practice.

Reply:
The second hardest problem is CS strikes again. I'm fine with that proposal.

Reply:
Note to self that this requires creating some new directories too (think `mkdir -p`).